bin/cli.ml

```diff
@@ -183,7 +183,7 @@ let handle_error = function
   | Error _ as r ->
       Logs.on_error
         ~pp:(fun fmt (`Msg msg) ->
-          format_linebreaks msg |> Format.pp_print_string fmt)
+          format_linebreaks msg |> Format.pp_print_string fmt )
```

Comment:
@gpetiot isn't it weird that it adds a space before the closing parens but not before the opening one?